server/index.js

```diff
@@ -3,22 +3,15 @@ module.exports.register.attributes = {
   name: 'hoodie'
 }
 
-var path = require('path')
-var urlParse = require('url').parse
-
 var corsHeaders = require('hapi-cors-headers')
 var hoodieServer = require('@hoodie/server').register
-var log = require('npmlog')
 var PouchDB = require('pouchdb-core')
```

Comment:
remove `PouchDB`, it will now be passed as `options.PouchDB` to the `register` function below